exotica_core/src/setup.cpp

```diff
@@ -107,7 +107,7 @@ std::vector<std::string> Setup::getSolvers() { return Instance()->solvers_.getDe
 std::vector<std::string> Setup::getProblems() { return Instance()->problems_.getDeclaredClasses(); }
 std::vector<std::string> Setup::getMaps() { return Instance()->maps_.getDeclaredClasses(); }
 std::vector<std::string> Setup::getCollisionScenes() { return Instance()->scenes_.getDeclaredClasses(); }
-Setup::Setup() : solvers_("exotica", "exotica::MotionSolver"), maps_("exotica", "exotica::TaskMap"), problems_(PlanningProblem_fac::Instance()), scenes_("exotica", "exotica::CollisionScene")
+Setup::Setup() : solvers_("exotica_core", "exotica::MotionSolver"), maps_("exotica_core", "exotica::TaskMap"), problems_(PlanningProblem_fac::Instance()), scenes_("exotica_core", "exotica::CollisionScene")
```

Comment:
I don't think we want to change this one - I'd prefer the namespace for plugins to remain `exotica`.